app/src/main/java/org/mozilla/focus/FocusApplication.kt

```diff
@@ -6,18 +6,24 @@ package org.mozilla.focus
 
 import android.os.StrictMode
 import android.preference.PreferenceManager
-import org.mozilla.focus.browser.UserAgent
+import android.support.annotation.VisibleForTesting
+import android.webkit.WebSettings
 import org.mozilla.focus.locale.LocaleAwareApplication
 import org.mozilla.focus.search.SearchEngineManager
 import org.mozilla.focus.session.VisibilityLifeCycleCallback
 import org.mozilla.focus.telemetry.TelemetryWrapper
 import org.mozilla.focus.utils.AppConstants
 import org.mozilla.focus.utils.OkHttpWrapper
 
-class FocusApplication : LocaleAwareApplication() {
+open class FocusApplication : LocaleAwareApplication() {
```

Comment:
Optional nit: do you think it might be a good idea to add "// Open for testing"?

Reply:
I figured when someone removed it and the tests broke, they'd know. @Baron-Severin What do you think?